Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchable.cs

```diff
@@ -76,6 +76,11 @@ public void SetLocalUp(Vector3 newLocalUp)
 
         protected new void OnValidate()
         {
+            if (Application.isPlaying)
+            {   // Don't validate during play mode
```

Comment:
Why don't we want to validate this one during play mode, but it's okay to do this in the components?

Reply:
In play mode the values being validated here are automatically checked by the methods you use to set them (eg SetLocalForward). So OnValidate is only necessary if you're tweaking the values by hand in the editor.